packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -864,19 +867,19 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
         while (!scope.canRequestFocus) {
           scope = scope.enclosingScope ?? _manager?.rootScope;
         }
-        scope?._doRequestFocus(findFirstFocus: false);
+        scope._doRequestFocus(findFirstFocus: false);
```

Comment:
Are these changes here related to the fix, or just cleaning up something? I assume scope can no longer be null when it might have before?

Reply:
Actually, I made those changes thinking that these couldn't ever be null, because the `rootScope` can't be null, but if the `_manager` is null, it still could be.  I reverted these.